pwndbg/lib/regs.py

```diff
@@ -4,12 +4,14 @@
 """
 from __future__ import annotations
 
-import collections
+from collections import OrderedDict
 from typing import Any
 from typing import Dict
 from typing import Iterator
 from typing import Tuple
 
+BitFlags = OrderedDict[str, int | Tuple[int, int]]
```

Comment:
Apparently this causes a TypeError on Ubuntu 20.04 :/

Reply:
I fixed it, the fix is weird to me because I'm using `typing.OrderedDict` as both a type hint and a constructor, but if it works it works.